### ISSUE TITLE
Anime filename support

### ISSUE THREAD

**Issue by breadcat** [jpillora/media-sort#18]
Hello, me again.

I'm having an issue sorting some anime filenames, this has happened a couple of times and is caused by the naming convention some anime releases have.

An example filename would be:
```
[Hakata Ramen] Dr. Stone (Season 1) [1080p x265]/[Hakata Ramen] Dr. Stone 12 [1080p x265].mkv
```

The releaser in brackets and the lack of series number in the filename is causing the lookup to be incorrect.
If I remove the releaser and add a series number with `mmv "\[Hakata Ramen\] Dr. Stone *" "Dr. Stone S01E#1"`  the detection works fine.

**Comment by jpillora** [comment]
That’s a tough one, so the season number is in the parent directory, and
the episode number is in the file name (12)?

**Comment by breadcat** [comment]
Yes, that's correct. If you were to give it a more traditional 'spin' it would be S01E12.
